RicOneApi/.editorconfig

```diff
@@ -0,0 +1,124 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
```

Comment:
Why to merge in master

Reply:
This project doesn't follow our branching structure since it's a fork of an open source project b y the New York Department of Education.